explorer/ast/paren_contents.h

```diff
@@ -26,13 +26,13 @@ namespace Carbon {
 // either `Expression` or `Pattern`.
 template <typename Term>
 struct ParenContents {
+  std::vector<Nonnull<Term*>> elements;
+  bool has_trailing_comma;
+
   // If this object represents a single term with no trailing comma, this
   // method returns that term. This typically means the parentheses can be
   // interpreted as grouping.
   auto SingleTerm() const -> std::optional<Nonnull<Term*>>;
```

Comment:
To be clear, I think this change should be undone due to the factory function.

Reply:
That's not a factory function, it's an accessor.